forward/fwd.go

```diff
@@ -327,8 +327,10 @@ func (f *httpForwarder) serveWebSocket(w http.ResponseWriter, req *http.Request,
 	}
 	go replicate(targetConn, underlyingConn, "backend", "client")
 	go replicate(underlyingConn, targetConn, "client", "backend")
-	err = <-errc
-	log.Infof("vulcand/oxy/forward/websocket: websocket proxying complete: %v", err)
+	err = <-errc // One goroutine complete
+	log.Infof("vulcand/oxy/forward/websocket: > websocket proxying complete: %v", err)
+	err = <-errc // Both goroutines complete
+	log.Infof("vulcand/oxy/forward/websocket: < websocket proxying complete: %v", err)
```

Comment:
Would saying "first direction/second direction proxying complete" be less confusing? As you notice the events might arrive in any order, and the channel doesn't carry the information about which direction has finished.